app/views/exports/_notification_pane.html.haml

```diff
@@ -18,4 +18,6 @@
     %a.btn.btn-primary{ href: '#export-options', data: { toggle: "tab" } }
       %i.fa.fa-arrow-left
       Back
-    = f.submit 'Export', class: 'btn btn-primary pull-right'
\ No newline at end of file
+    = f.submit 'Export', class: 'btn btn-primary pull-right'
+    = link_to root_path(params: current_search.params), class: 'btn btn-default pull-right', style: 'margin-right: 10px;' do
```

Comment:
No need to pass the `current_search.params` as part of the link.  The system will cached the current search params into the session and they get loaded automatically when viewing the map page.